### PR TITLE
Update CSV read/write BBE

### PR DESCRIPTION
**Pull request by sahanHe**
## Purpose
Update CSV read/write BBE as a fix for [#4728](https://github.com/ballerina-platform/ballerina-dev-website/issues/4728).

### CODE REVIEW

File: examples/io-csv/io_csv.bal [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
-    // Initializes the CSV file path and content.
+    // Initializes the CSV file paths and content.
     string csvFilePath1 = "./files/csvFile1.csv";
     string csvFilePath2 = "./files/csvFile2.csv";
     string[][] csvContent = [["1", "James", "10000"], ["2", "Nathan", "150000"],
     ["3", "Ronald", "120000"], ["4", "Roy", "6000"],
     ["5", "Oliver", "1100000"]];
 
-    // Writes the given content string[][] to a CSV file.
+    // Writes the given content string[][] to a CSV file in csvFilePath1.
```
Comment (daneshk) [inline examples/io-csv/io_csv.bal:11]:
```suggestion
    // Writes the given content string[][] to a CSV file.
```

File: examples/io-csv/io_csv.bal [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
     check io:fileWriteCsv(csvFilePath1, csvContent);
-    // If the write operation was successful, then, performs a read operation to read the CSV content as a string array of arrays.
+    // Reads the previously saved CSV file in csvFilePath1 as a string[][], 
+    // if the write operation was successful.
```
Comment (daneshk) [inline examples/io-csv/io_csv.bal:14]:
```suggestion
    // Reads the CSV file(the previously saved file) as a string[][].
```

File: examples/io-csv/io_csv.bal [hunk 3]
```diff
@@ -1,26 +1,28 @@
...
     string[][] readCsv = check io:fileReadCsv(csvFilePath1);
     io:println(readCsv);
 
-    // Writes the given content stream to a CSV file.
+    // Writes the given content as a stream to a CSV file in csvFilePath2.
     check io:fileWriteCsvFromStream(csvFilePath2, csvContent.toStream());
-    // If the write operation was successful, then, perform a read operation to read the CSV content as a stream.
+    // Reads the CSV file in csvFilePath2 as a stream,
+    // if the previous step of writing a CSV file from stream was successful.
```
Comment (daneshk) [inline examples/io-csv/io_csv.bal:21]:
```suggestion
    // Reads the CSV file(the previously saved file) as a stream.
```

File: examples/io-csv/io_csv.bal [hunk 4]
```diff
@@ -10,13 +10,13 @@ public function main() returns error? {
 
     // Writes the given content string[][] to a CSV file.
     check io:fileWriteCsv(csvFilePath1, csvContent);
-    // If the write operation was successful, then, performs a read operation to read the CSV content as a string array of arrays.
+    // Reads the CSV file(the previously saved file) as a string[][].
```
Comment (praneesha) [inline examples/io-csv/io_csv.bal:13]:
```suggestion
    // Reads the previously-saved CSV file as a `string[][]`.
```

File: examples/io-csv/io_csv.bal [hunk 5]
```diff
@@ -10,13 +10,13 @@ public function main() returns error? {
...
     string[][] readCsv = check io:fileReadCsv(csvFilePath1);
     io:println(readCsv);
 
-    // Writes the given content stream to a CSV file.
+    // Writes the given content as a stream to a CSV file.
     check io:fileWriteCsvFromStream(csvFilePath2, csvContent.toStream());
-    // If the write operation was successful, then, perform a read operation to read the CSV content as a stream.
+    // Reads the CSV file(the previously saved file) as a stream.
```
Comment (praneesha) [inline examples/io-csv/io_csv.bal:19]:
```suggestion
    // Reads the CSV file(the previously saved file) as a stream.
```
Same as above.

**Review by praneesha — APPROVED** [review]

**Pull request merged**